src/pages/events/module-development.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Commerce module development
+description: Learn what you need to do to enable your modules for Adobe I/O Events.
+---
+
+# Commerce module development
+
+This topic describes how to enable your custom modules for Adobe I/O Events. You can also manually register observer events using the [`events:subscribe` command](./commands.md#subscribe-to-a-commerce-event).
+
+## Register events
+
+You can programmatically register events using the following methods:
+
+*  Create an `io_events.xml` file in your module
+*  Declare them in the system `config.php` file
+
+### io_events.xml
+
+Create the `<module-root>/app/etc/io_events.xml` file and define a list of events that should always be emitted. Events listed in this file cannot be disabled with the [`events:unsubscribe` command](./commands.md#unsubscribe-from-a-commerce-event).
```

Comment:
Please remove `app/` from the path, it should be `<module_root>/etc/io_events.xml`